pkg/http/api.go

```diff
@@ -76,17 +73,14 @@ type API interface {
 }
 
 type api struct {
-	universal                  *universalapi.UniversalAPI
+	*universalapi.UniversalAPI
```

Comment:
Why this change? Not sure we should have `api` depend on this, since all methods would automatically be exported and if we have a method with the same name, but different arguments, there'd be a conflict IIRC

Reply:
This was to keep it consistent with `grpc`, but happy to revert.